Update azure-dns.md

laser84 opened the pull request:
Better solution for **CentOS by Rogue Wave Software (formerly OpenLogic)** (uses NetworkManager):
1. Add 'RES_OPTIONS="timeout:1 attempts:5"' to '/etc/sysconfig/network'.

azurecla commented:
Hi __@charislove__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@charislove 
This is a public repository, and anyone can read it.
Please submit this article to azure-content-pr in the release-build-day-1 branch.

Commit:
Update azure-dns.md

Example **CentOS by Rogue Wave Software (formerly OpenLogic)** (uses NetworkManager) not work in CentOS 7.
Solution:
1. Add 'RES_OPTIONS="timeout:1 attempts:1"' to '/etc/sysconfig/network'.
2. Run 'systemctl restart NetworkManager' to update.

Commit:
Update azure-dns.md

Commit:
Update azure-dns.md
